### DISCUSSION TITLE
How to persist data on ECS

### DISCUSSION THREAD

**Discussion by MoeDawood15** [nocodb/nocodb#7252] [Q&A]
Hi 

Hope all is well 

A quick question , how does data persist on ECS , once a task is restarted everything resets itself , does anyone know how data can be persisted via ECS ?

**Comment by o1lab** [comment]
what data are you refering to ? please [refer to env variables](https://docs.nocodb.com/getting-started/self-hosted/environment-variables/) NC_DB and *s3* variables to persists databases and files.

**Comment by MoeDawood15** [comment]
@o1lab  are there any plans to add this to the installation guide for ECS (Fargate) ?
